visual-recognition/v3.ts

```diff
@@ -127,8 +127,8 @@ class VisualRecognitionV3 extends BaseService {
         },
         'url': _params.url,
         'threshold': _params.threshold,
-        'owners': Array.isArray(_params.owners) ? _params.owners.join(',') : _params.owners,
-        'classifier_ids': Array.isArray(_params.classifierIds) ? _params.classifierIds.join(',') : _params.classifierIds
```

Comment:
These need to manually be changed back to normal after generating, I forgot to go over that with you